kornia/enhance/adjust.py

```diff
@@ -747,6 +749,32 @@ def equalize3d(input: torch.Tensor) -> torch.Tensor:
     return torch.stack(res)
 
 
+def invert(input: torch.Tensor, max_val: torch.Tensor = torch.tensor(1.0)) -> torch.Tensor:
```

Comment:
This currently supports any shaped tensors. Shall we restrict it to be at least 2D?